src/components/RaceCards/RaceCard.module.scss

```diff
@@ -0,0 +1,50 @@
+@import '../../styles/theme/variables.scss';
+
+// Card slide up controls
+.cardContainer {
+    overflow-y: hidden;
+    display: flex;
+    flex-direction: column;
+    flex: 0 0 25%;
+    max-width: 375px;
+    height: 300px;
+    margin: 0.25rem;
+    background-color: black;
+    border-radius: 0.25rem;
+    .cardTitle {
+        position: relative;
+        top: 240px;
+        background-color: rgba(0, 0, 0, 0.65);
```

Comment:
Make this transparancy a var to use in our styles and import it instead as part of our theme